docs/pages/accounts/sso.mdx

```diff
@@ -69,6 +78,8 @@ If you wish to remove them ahead of that time or you wish to remove them to clea
 
 An active Enterprise Plan is required to continue using SSO. [Contact us](https://expo.dev/contact) if you wish to discontinue the use of SSO or change your plan.
 
+In order to ensure uninterrupted access to your organization whether or not SSO is enabled, SSO organizations must keep at least one non-SSO user with the Owner role as a member.
```

Comment:
I don't think we've ever verified this. In theory, I think a SSO org could replace all their regular users with SSO users. It just would theoretically add risk of losing access if the IdP disabled all users.

Reply:
Yeah, I doubt it breaks anything as long as SSO access is maintained, and I'm not aware of us enforcing this (though maybe we should?). I just want to set the expectation that they should keep this one account even after they're otherwise using SSO 100%, because there's no clear path to recovery if SSO suddenly cannot be used for any number of reasons (non-payment, IdP token is revoked and the person who has access to generate a new one is on vacation, etc. etc)